src/MyParcelComApi.php

```diff
@@ -837,17 +839,38 @@ private function arrayToFilter(array $array)
     {
         $filters = [];
         foreach ($array as $name => $value) {
-            /**
-             * @deprecated This condition is to keep the SDK backwards
-             * compatible until the API has been updated to use the new filter.
-             */
-            if ($name === 'has_active_contract') {
-                $filters['filter[has_contract]'] = $value;
-            }
-
             $filters["filter[$name]"] = $value;
         }
 
         return $filters;
     }
+
+    /**
```

Comment:
I think some sort of explanation on what does method does would be welcome. Specifically the `$specificCarrier` parameter isn't really clear to me.